[2025-02-23T15:52:39.580+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:39.602+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-12T00:00:00+00:00 [queued]>
[2025-02-23T15:52:39.614+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-12T00:00:00+00:00 [queued]>
[2025-02-23T15:52:39.615+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:39.632+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-12 00:00:00+00:00
[2025-02-23T15:52:39.640+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1501) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:39.642+0000] {standard_task_runner.py:72} INFO - Started process 1513 to run task
[2025-02-23T15:52:39.640+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-12T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpmzhyxo_v']
[2025-02-23T15:52:39.643+0000] {standard_task_runner.py:105} INFO - Job 47: Subtask third_task
[2025-02-23T15:52:39.687+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-12T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:39.768+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-12T00:00:00+00:00'
[2025-02-23T15:52:39.769+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:39.781+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:39.783+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-23T15:52:39.790+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:39.791+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-23T15:52:39.792+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:39.814+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:39.815+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-12T00:00:00+00:00, execution_date=20250212T000000, start_date=20250223T155239, end_date=20250223T155239
[2025-02-23T15:52:39.857+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:39.871+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:39.873+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:15.616+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:15.653+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-12T00:00:00+00:00 [queued]>
[2025-02-24T04:47:15.681+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-12T00:00:00+00:00 [queued]>
[2025-02-24T04:47:15.681+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:15.743+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-12 00:00:00+00:00
[2025-02-24T04:47:15.755+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=948) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:15.760+0000] {standard_task_runner.py:72} INFO - Started process 970 to run task
[2025-02-24T04:47:15.760+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-12T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp046f_m14']
[2025-02-24T04:47:15.772+0000] {standard_task_runner.py:105} INFO - Job 46: Subtask third_task
[2025-02-24T04:47:15.891+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-12T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:16.007+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-12T00:00:00+00:00'
[2025-02-24T04:47:16.010+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:16.030+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:16.032+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-24T04:47:16.048+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:16.057+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-24T04:47:16.059+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:16.099+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:16.100+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-12T00:00:00+00:00, execution_date=20250212T000000, start_date=20250224T044715, end_date=20250224T044716
[2025-02-24T04:47:16.337+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:16.393+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:16.397+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
